task-tracker-console-application/src/main/java/com/isa/user/Login.java

```diff
@@ -0,0 +1,135 @@
+package com.isa.user;
+
+import com.isa.menu.MainMenu;
```

Comment:
jak zmieniłaś nazwę pakietu to też w miejscach gdzie było używane trzeba to uwzględnić, bo mi `Main` nie działa ;)

Reply:
W najnowszym commicie uwzględniłam klasę Main ze zmianą :)